fix: change SBOM route to PUT

patheard opened the pull request:
# Summary
By default, the Dependency Track `/api/v1/bom` route
is expecting a PUT request with a base64 encoded BOM.

# Related
* cds-snc/platform-sre-security-support#94

patheard commented on images/asset_inventory/cartography/Dockerfile:
```diff
@@ -0,0 +1,24 @@
+COPY --chown=python:python requirements.txt /home/python/cartography/requirements.txt
+RUN /home/python/venv/bin/pip install --no-cache-dir --requirement /home/python/cartography/requirements.txt
+
+USER root
+COPY --chown=python:python docker-entrypoint.sh /
+RUN chmod +x /docker-entrypoint.sh
+RUN chown -R python:python /config
+USER python
```
Could this block be moved about the `USER python` on line 10 so you didn't have to switch back and forth between root and python?

mohamed-cds replied:
Might be from copy pasting to success. I've modified it to group the root actions 🎉

patheard committed:
fix: change SBOM route to PUT

By default, the Dependency Track /api/v1/bom route
is expecting a PUT request with a base64 encoded BOM.

mohdnr reviewed (approved):
LGTM 🚀 ✅
